packages/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.ts

```diff
@@ -1,38 +1,441 @@
 import { SlotClassNames } from '@fluentui/react-utilities';
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SpinButtonSlots, SpinButtonState } from './SpinButton.types';
+import { tokens } from '@fluentui/react-theme';
+import { useInputStyles_unstable } from '@fluentui/react-input';
 
-/**
- * @deprecated Use `spinButtonClassNames.root` instead.
- */
-export const spinButtonClassName = 'fui-SpinButton';
 export const spinButtonClassNames: SlotClassNames<SpinButtonSlots> = {
   root: 'fui-SpinButton',
   input: 'fui-SpinButton__input',
-  incrementControl: 'fui-SpinButton__incrementControl',
-  decrementControl: 'fui-SpinButton__decrementControl',
+  incrementButton: 'fui-SpinButton__incrementButton',
+  decrementButton: 'fui-SpinButton__decrementButton',
 };
 
-/**
- * Styles for the root slot
- */
-const useStyles = makeStyles({
-  root: {
-    // TODO Add default styles for the root element
+const spinButtonExtraClassNames = {
+  button: 'fui-SpinButton__button',
```

Comment:
if the only use of this is adding `_active` to it, should this classname itself just be `activeButton`?

Reply:
My thinking is that it's nice to have a general "button" class name selector for both the increment and decrement buttons but it looks like I never added it to the class list. I like your idea better.